src/activities/content/ContentEntity.js

```diff
@@ -65,4 +65,12 @@ export class ContentEntity extends Entity {
 	getHtmlFileHref() {
 		return ContentHelperFunctions.getHrefFromRel(Rels.Content.htmlFileEntity, this._entity);
 	}
+
+	/**
+	 * @returns {string} lesson-view-page link
```

Comment:
Am i right to assume that if this link won't always be added then the return type would be `{string|undefined}` (or null?) ? I know we've updated that kind of stuff in other spots but not here.

Reply:
It's not a big deal since we don't really _need_ this "typing". but to me I'd rather be accurate or just not even include it at all.

Reply:
You are correct, I just went with the typing of the other functions using the same helper. I think it can also return `null` (rather than `undefined`). Here is the implementation of the helper function:

```
getHrefFromRel: (rel, entity) => {
		return entity
			&& entity.hasLinkByRel(rel)
			&& entity.getLinkByRel(rel).href;
	}
```

Reply:
Resolved in https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/313/commits/56f165dfdefe5ef26b2d4ba4d399f39869804d18